lib/smbios.c

```diff
@@ -572,10 +621,36 @@ ulong write_smbios_table(ulong addr)
 
 	ctx.node = ofnode_null();
 	if (IS_ENABLED(CONFIG_OF_CONTROL)) {
+		int ret;
+
 		uclass_first_device(UCLASS_SYSINFO, &ctx.dev);
-		if (ctx.dev)
+		if (ctx.dev) {
+			printf("=====================> ctx.dev->name: %s\n", ctx.dev->name);
+
```

Comment:
Please rebase it on the top of this
lore.kernel.org/r/4e7e75bae7104bfca3964f7a442cac9f084a1f5b.1714138688.git.michal.simek@amd.com